test/transforms/populate.js

```diff
@@ -14,7 +14,8 @@ describe('transforms/populate', function () {
                 followRelations: {type: query.not('HtmlStyle')}
             });
 
-        expect(assetGraph, 'to contain no assets', 'Css');
+        expect(assetGraph, 'to contain asset', 'Css');
+        expect(assetGraph, 'to contain asset', { type: 'Css', isLoaded: false });
```

Comment:
This seems to just be a more specific version of the previous assertion. Kill the less specific version?

Reply:
You're right, fixed! It was late :)